.github/workflows/java_tests.yml

```diff
@@ -152,46 +127,3 @@ jobs:
         with:
           name: java_wordcount_direct_runner-${{matrix.os}}
           path: examples/java/build/reports/tests/integrationTest
-
-  java_wordcount_dataflow:
```

Comment:
We already have GitHub workflow that test Java WordCount on Dataflow. It was skipped for so long and seems just another thing to maintain in our codebase.